.github/workflows/build_python_3.yml

```diff
@@ -87,6 +87,7 @@ jobs:
               mv $w {dest_dir}
             done &&
             rm -rf ./tempwheelhouse
+          CIBW_ENVIRONMENT_MACOS: "MACOSX_DEPLOYMENT_TARGET=12"
```

Comment:
Do you know whether setting this closer to setup.py would help native tools pick up minimum required dependencies?  As far as I can tell, this parameter is inferred from the environment in a lot of cases.  If we were more strict, we may not have to set it here.

... it's also possible that we've just hit the limit of what we can accomplish with these legacy OSX targets and our only option is to limit what we support.  Any clue how old "12" is versus the old target (10.9)?

Reply:
Don't mind this PR, I didn't see yours merged 🙈

Reply:
Oh.  Well, I think we'll have to do a PR like this sometime anyway, so getting runtime versions sorted out is a good thing.  :)